server/routes/index.ts

```diff
@@ -60,21 +60,24 @@ export function defineRoutes(router: IRouter, esClient: IClusterClient) {
   });
 
   const roleMappingSchema = schema.object({
-    description: schema.string(),
+    description: schema.maybe(schema.string()),
     backend_roles: schema.arrayOf(schema.string()),
     hosts: schema.arrayOf(schema.string()),
     users: schema.arrayOf(schema.string()),
   });
 
   const roleSchema = schema.object({
-    description: schema.string(),
+    description: schema.maybe(schema.string()),
     cluster_permissions: schema.nullable(schema.arrayOf(schema.string())),
     tenant_permissions: schema.arrayOf(schema.any()),
     index_permissions: schema.arrayOf(schema.any()),
   });
 
   const tenantSchema = schema.object({
-    description: schema.string(),
+    description: schema.maybe(schema.string()),
+    reserved: schema.boolean(),
+    hidden: schema.boolean(),
+    static: schema.boolean(),
```

Comment:
I don't think reserved, hidden and static is determined by user.

Reply:
agreed